WordPress/src/test/java/org/wordpress/android/ui/reader/usecases/ReaderGetPostUseCaseTest.kt

```diff
@@ -0,0 +1,111 @@
+package org.wordpress.android.ui.reader.usecases
+
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.spy
+import com.nhaarman.mockitokotlin2.whenever
+import kotlinx.coroutines.InternalCoroutinesApi
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mock
+import org.wordpress.android.BaseUnitTest
+import org.wordpress.android.TEST_DISPATCHER
+import org.wordpress.android.datasets.wrappers.ReaderPostTableWrapper
+import org.wordpress.android.models.ReaderPost
+import org.wordpress.android.models.ReaderPostDiscoverData
+import org.wordpress.android.models.ReaderPostDiscoverData.DiscoverType
+import org.wordpress.android.test
+
+@InternalCoroutinesApi
+class ReaderGetPostUseCaseTest : BaseUnitTest() {
+    @Mock private lateinit var readerPostTableWrapper: ReaderPostTableWrapper
+    private val postId = 1L
+    private val blogId = 2L
```

Comment:
Suggestion (💡): What if those ids are named to:
```
    private val readerPostId = 1L
    private val readerBlogId = 2L
    private val discoverPostId = 3L
    private val discoverBlogId = 4L
```

And then used as:
```
    private val readerPost = ReaderPost().apply {
        postId = readerPostId
        blogId = readerBlogId
    }

    private val readerDiscoverSourcePost = ReaderPost().apply {
        postId = discoverPostId
        blogId = discoverBlogId
    }
```

Just to make it more visible what is what and avoid doing the `this@ReaderGetPostUseCaseTest` trick?

Reply:
Nice 👍, in 843e773